<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>intru</class>
 <widget class="QWidget" name="intru">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Индивидуальная тренировка</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>761</width>
     <height>401</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QTableWidget" name="twIndU">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>721</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="teIndU">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>711</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBackIndU">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>290</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConnect">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>290</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Подключение</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
